pages/admin/programs/index.tsx

```diff
@@ -0,0 +1,146 @@
+import { AdminEmptyState } from "@components/admin/AdminEmptyState";
+import { useState } from "react";
+import { roles } from ".prisma/client";
+import { AdminHeader } from "@components/admin/AdminHeader";
+
+type BrowseProgramsProps = {
+    session: Record<string, unknown>;
+};
+
+const headerLinks = [
+    { name: "Add Program", url: "/admin/program/create" },
+    { name: "Add Class", url: "/admin/class/create" },
+];
+
+export const BrowsePrograms: React.FC<BrowseProgramsProps> = (props) => {
+    const router = useRouter();
+    const [searchTerm, setSearchTerm] = useState("");
+
+    const {
+        programs: programCardInfos,
+        isLoading,
+        error,
+    } = usePrograms(router.locale as locale);
+
+    if (isLoading) {
+        return <Loading />;
+    } else if (error) {
+        return <Box>An Error has occurred</Box>;
+    }
+
+    const filteredCards = programCardInfos.filter((prog) => {
+        const term = searchTerm.toLowerCase();
+        if (term == "") {
+            return prog;
+        } else if (
+            prog.name.toLowerCase().includes(term) ||
+            prog.description.toLowerCase().includes(term) ||
+            prog.onlineFormat.toLowerCase().includes(term) ||
+            prog.tag.toLowerCase().includes(term)
+        ) {
+            return prog;
+        }
+    });
+    return (
+        <Wrapper session={props.session}>
+            <Box>
+                <AdminHeader headerLinks={headerLinks}>Programs</AdminHeader>
+
+                <Box px="50px">
+                    <Text fontSize="16px">Browse Programs</Text>
+                    <InputGroup mt="25px">
+                        <InputLeftElement
+                            pointerEvents="none"
+                            children={<SearchIcon color="gray.300" />}
+                        />
+                        <Input
+                            pl={8}
+                            placeholder={"Search Programs"}
+                            onChange={(e) => {
+                                setSearchTerm(e.target.value);
+                            }}
+                        />
+                    </InputGroup>
+                </Box>
+
+                <Box ml="50px" mt="25px">
+                    {filteredCards && filteredCards.length > 0 ? (
+                        <Grid templateColumns="repeat(3, 1fr)" gap={6}>
+                            {filteredCards.map((item, idx) => {
+                                return (
+                                    <GridItem key={idx}>
+                                        <BrowseProgramCard cardInfo={item} />
+                                    </GridItem>
+                                );
+                            })}
+                        </Grid>
+                    ) : (
+                        <Box pr="50px">
+                            <AdminEmptyState
+                                w="100"
```

Comment:
is this suppose to be 100%?
```suggestion
                                w="100%"
```